### DISCUSSION TITLE
Something wrong with PeleMP/PP_Spray/BreakupSplash module

### DISCUSSION THREAD

**Discussion by RachelMeng0096** [AMReX-Combustion/PeleC#721] [Q&A] [answered]
Greetings everyone, I encountered the following error while compiling the test file for Spray in PeleMP. As a novice in Pele, I am unsure how to resolve this issue. Could anyone kindly assist me?
![image](https://github.com/AMReX-Combustion/PeleC/assets/153271449/1adf3318-f177-41f0-af43-04e5663e969e)

**Comment by marchdf** [comment, marked as answer]
Hi! thank you for reaching out. I would actually recommend that you pull the latest version of PeleC (containing the commit I merged this morning). We did a big reorg of pelemp (moved everything to pelephysics) and fixed a bug in PeleMP at the same time. My guess is that it should fix your problem as well. Make sure you do a `git submodule update` after pulling the latest version. Let us know if this works out for you!

**Reply by baperry2** [reply]
With the way we've restructured the submodules, you may actually have to do `git submodule update --init --recursive`

**Reply by RachelMeng0096** [reply]
Thank you kindly for your response. I shall proceed to download the latest version.
It appears that the issue may stem from inaccuracies in the Make.PeleC file, as it lacks inclusion of the BreakupSplash's Make.package. Having attempted an addition, the situation seems promising thus far.
![image](https://github.com/AMReX-Combustion/PeleC/assets/153271449/34c9c849-26d9-4300-bc46-640bd408e1e9)

**Reply by marchdf** [reply]
yes that is indeed one of the things we fixed in the new version.
